src/components/Button/Button.scss

```diff
@@ -63,6 +63,10 @@
 	}
 }
 
+a.ms-Button {
```

Comment:
It should also remove it if we add `text-decoration: none;` directly to `.ms-Button`, right? We try to avoid targeting specific elements wherever possible.